elide-model-config/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/jsonformats/ElideRSQLFilterFormatAttr.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers.jsonformats;
+
+import com.github.fge.jackson.NodeType;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import com.github.fge.jsonschema.core.report.ProcessingReport;
+import com.github.fge.jsonschema.format.AbstractFormatAttribute;
+import com.github.fge.jsonschema.processors.data.FullData;
+import com.github.fge.msgsimple.bundle.MessageBundle;
+
+public class ElideRSQLFilterFormatAttr extends AbstractFormatAttribute {
+    private static final String RSQL_FILTER_FORMAT_REGEX = "^[A-Za-z][0-9A-Za-z_]*(==|!=|>=|>|<|<=|=[a-z]+=)(.*)$";
```

Comment:
I wonder if we can do something stronger here.  I wonder if we can parse the expression with FIQL/RSQL parser.  A regex is not expressive enough to capture all the legal values.